modules/migration-performance-metrics.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc.adoc
+
+[id="migration-performance-metrics_{context}"]
+= Performance metrics
+
+The Migration controller records a set of metrics and pulls it into on-cluster monitoring storage. You can query the metrics via PromQL for diagnosing migration performance issues. All metrics reset when the Migration controller manager container restarts.
+
+== Provided Metrics
+
+=== cam_app_workload_migrations
+
+This metric is a count of MigMigration CRs over time. It is useful for viewing alongside `mtc_client...` metrics to collate API request info with migration status changes. This metric is included in Telemetry.
+
+[%header,cols="3,3,3"]
+|===
+|Queryable Label Name |Sample Label Values |Label Description
+
+|status
+|`running`, `idle`, `failed`, `completed`
+|Status of MigMigration CR
+
+|type
+|`stage`, `final`
+|Type of MigMigration CR
+|===
+
+=== mtc_client_request_count
+
+This metric is a cumulative count of k8s API requests issued from the Migration controller. It is not included in Telemetry.
+
+[%header,cols="3,3,3"]
+|===
+|Queryable Label Name |Sample Label Values |Label Description
+
+|cluster
+|`\https://migcluster-url:443`
+|Cluster request was issued against
+
+|component
+|`migplan`, `migcluster`
+|Sub-controller API request was issued from
+
+|function
+|`(*ReconcileMigPlan).Reconcile`
+|Function request was issued from
+
+|kind
+|`SecretList`, `Deployment`
+|K8s kind request was issued for
+|===
+
+
+=== mtc_client_request_elapsed
+
+This metric is a cumulative latency (in milliseconds) of k8s API requests issued from the Migration controller. Not included in Telemetry. {table same as above?}
+
+[%header,cols="3,3,3"]
+|===
+|Queryable Label Name |Sample Label Values |Label Description
+
+|cluster
+|`\https://migcluster-url:443`
+|Cluster request was issued against
+
+|component
+|`migplan`, `migcluster`
+|Sub-controller API request was issued from
+
+|function
+|`(*ReconcileMigPlan).Reconcile`
+|Function request was issued from
+
+|kind
+|`SecretList`, `Deployment`
+|K8s kind request was issued for
+|===
+
+
+== Accessing Metrics in the {product-title} 4 Monitoring UI
+
+. From the {product-title} 4 web console, select the sidebar menu item Monitoring -> Metrics.
```

Comment:
UI labels need to be marked. See OCP doc guidelines.
Check the term "sidebar menu item". This does not sound right.